randovania/data/json_data/prime3.txt

```diff
@@ -3812,12 +4678,12 @@ Asset id: 1808913861159744919
 Command Courtyard
```

Comment:
- Add two new events to this room
  - "Command Courtyard Grapple Voltage Barrier"
  - "Command Courtyard Fans"
- Remove the two helper nodes